### DISCUSSION TITLE
Unreal missing Chromium and gameoverlayrenderer dsym files

### DISCUSSION THREAD

**Discussion by ellishawn** [getsentry/support#9] [Support] [answered]
I'm trying to get Sentry crash reporter working with the Mac version of our game and I keep running into these two problems when I cause a crash on the Mac:

A required debug information file was missing. 
ba2b8168-3fa2-3440-8823-b794caffbfd5 (for Chromium Embedded Framework)
1
a few seconds ago
A required debug information file was missing. 
bbe307c9-fbf5-3d10-b40c-457f951277b3 (for gameoverlayrenderer.dylib)
1
a few seconds ago

I can't find these dsym files anywhere on my Mac or the internet, can anyone help point me in the right direction?

**Comment by bruno-garcia** [comment, marked as answer]
Isn't that being shipped with your game? I imagine it would be part of the build output
